### DISCUSSION TITLE
RNode, a New Model

### DISCUSSION THREAD

**Discussion by markqvist** [markqvist/Reticulum#64] [Show and tell]
I will have a more detailed post up on [unsigned.io](https://unsigned.io) soon, but felt like sharing a bit here first.

The chip supply issues and various other factors was a pretty good motivation for a complete refresh of the RNode platform, and I have been hard at work behind the scenes preparing this, both software and hardware wise. So much goes into this, and I am quite excited to get this ball rolling soon.

There is going to be *a lot* of upgrades to the platform as a whole, both in terms of functionality of the device, ease of use, integration with Reticulum, but also in terms of deployment of these devices at scale.

I will still be producing and supplying RNodes at a limited scale, but with this new system, distributed production by anyone who wishes to make and sell RNodes will be highly encouraged. I think this is essential to strengthening the availability of resilient communications tools.

The goal is that anyone who wants to build and sell RNodes locally will be able to do so, and with very minimal hassle, and practically zero starting capital. Another goal is that many different types and flavors of RNodes can exist, each adapted to the creators and users needs. If you get what I am getting at here, and you already know this is something you want to get involved in, let me know over LXMF or Matrix.

More on all of this soon :) Now, for a few pictures:

![0](https://user-images.githubusercontent.com/1438012/172575557-8d56396a-7af5-48da-b737-818f57c4dadd.jpg)

![5](https://user-images.githubusercontent.com/1438012/172575906-b8fbe3db-a9ec-4352-bc82-f2c82815616a.jpg)

![12](https://user-images.githubusercontent.com/1438012/172575820-bc44279f-ae3e-48f2-8fcb-fc7d7954dc96.jpg)

**Comment by markqvist** [comment]
And just to be clear, the pictures above are some early prototype units. A lot will probably still change :)

**Comment by silveroranges** [comment]
Man I can't wait. The biggest issue I have had so far was the chip shortage, and the price gouging happening with raspberry pi's. I appreciate making it available for other people to produce, I look forward to the write-up on unsigned.io!

**Comment by trosel** [comment]
I like that you are thinking about this project so holistically. Nothing is overlooked. 

> The goal is that anyone who wants to build and sell RNodes locally will be able to do so, and with very minimal hassle, and practically zero starting capital. 

I'll move back to my hometown in the middle of nowhere for a couple of months to sell these / set them up lol

**Comment by OtisByron** [comment]
@markqvist I'm certainly excited about this! I've been trying to send a message via NomadNet but it doesn't seem to go through. I'll keep working on it. Just wanted to drop a line and say that I'm interested in getting involved with this and I'm working on getting in touch.

**Reply by markqvist** [reply]
Sorry, I was completely disconnected for a couple of days. It should have gotten picked up by a propagation node though, don't know why that did not happen for you. Maybe one was not available for your client?

Either way, I am definitely here now, and so is the post going into some more detail about all of this: https://unsigned.io/rnode-a-new-model/

**Comment by markqvist** [comment]
I just put up a post over on unsigned.io on the new model of RNode production for anyone who is interested: https://unsigned.io/rnode-a-new-model/

The TLDR is:
- I have been working on a new v2.x RNode firmware branch. It is coming out in the relatively near future.
- The goal is to widen hardware support, and make it possible build many kinds of RNodes, that can still interoperate with each other (obviously only when using the same modulation schemes and frequencies)
- The new platform supports multiple transceivers on the same RNode, and both wired and wireless
- Anyone can design, produce, and even sell their own particular flavors of RNodes

For more details, head over and read the post!

**Reply by githubques** [reply]
Can these be battery powered and placed on their own without a raspberry pi/laptop connection to be pinged on remotely like how I believe Meshtastic works (also uses T-Beams, etc.).  If so, what kind of battery will your design require? Still trying to find the right size of 18650 batteries for T-Beams I purchased in nearby stores.

**Reply by markqvist** [reply]
The current prototype you see above has a battery compartment in the case for standard 3.7v lithium batteries with a JST connector. The devices I will sell will be available with and without batteries, but you can also supply your own batteries in any capacity you would like. There is also charging circuitry on-board, so you can charge it from a small solar setup providing 5v to the RNode.

The v2 firmware will allow for various stand-alone operating modes :) There is a Remote Repeater function, for example, and some LXMF functionality is also coming, but wont be available on day one of release.

**Reply by githubques** [reply]
Awesome!
